### PR TITLE
Find an available slug if the requested one is in use

### PR DESCRIPTION
**Pull request by kaedroho**

### CODE REVIEW

File: wagtail_localize/models.py [hunk 1]
```diff
@@ -475,7 +476,7 @@ def create_or_update_translation(self, locale, user=None, publish=True, copy_par
 
                 if isinstance(translation, Page):
                     # Make sure the slug is valid
-                    translation.slug = slugify(translation.slug)
+                    translation.slug = find_available_slug(translation.get_parent(), slugify(translation.slug), ignore_page_id=translation.id)
```
Comment (kaedroho) [inline wagtail_localize/models.py:479]: TODO: Add ignore_page_id into Wagtail

Comment (kaedroho) [reply wagtail_localize/models.py:479]: https://github.com/wagtail/wagtail/pull/6412

**Review by KalobTaulien — APPROVED** [review]
Nice, I didn't know about `find_available_slug`!

**Comment by kaedroho** [comment]
Thanks!

**Pull request merged**